### ISSUE TITLE
accessibility plugin errors for nested menu

### ISSUE THREAD

**Issue by optisiddiqus** [optimizely-axiom/optiaxiom#1059]
for `Nested` and `Nested and Searchable` components, there are a few violations / errors showing from the accessibility plugin.

seems minor but we should still fix those if we can.

<img width="1726" alt="Image" src="https://github.com/user-attachments/assets/17443aed-b41f-4aef-b16d-2342f14d5301" />

**Comment by radnan** [comment]
thanks so much for reporting this! issue has been fixed in main - will release in the next patch version update.

will also check why our storybook isn't flagging these a11y issues in tests.
